### PR TITLE
INTMDB-594: [CFN-PI] Improve documentations for quickstarts

### PR DESCRIPTION
**Pull request by andreaangiolillo**
## Proposed changes
_Jira ticket:_ INTMDB-594

- Moved [master/quickstart-examples](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/quickstart-examples) to a folder under example
- Add a `README.md` for all the quickstarts
- Add `execution-role.yml` to all the quickstart to list the permission required to run the template

Screenshot 2023-03-31 at 13 53 19

## How to review this PR

- Reviews the README.md files
     - [quickstart-mongodb-atlas/README.md](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/INTMDB-594/examples/quickstart-mongodb-atlas/README.md)
     - [quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/INTMDB-594/examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md)
     - [quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md)](https://github.com/mongodb/mongodbatlas-cloudformation-resources/blob/INTMDB-594/examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md)

## Type of change:

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as
  expected)
- [ ] This change requires a documentation update

## Manual QA performed:

- [ ] cfn invoke for each of CRUDL/cfn test
- [ ] Updated resource in  [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples)
- [ ] Published to AWS private registry
- [ ] Used the template in [example](https://github.com/mongodb/mongodbatlas-cloudformation-resources/tree/master/examples) to create and update a stack in AWS
- [ ] Deleted stack to ensure resources are deleted
- [ ] Created multiple resources in same stack
- [ ] Validated in Atlas UI
- [ ] Included screenshots

## Required Checklist:

- [x] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation (if appropriate)
- [ ] I have run `make fmt` and formatted my code
- [ ] For CFN Resources: I have released by changes in the private registry and proved by change
  works in Atlas

### CODE REVIEW

**Comment by colm-quinn** [comment]
@jwilliams-mongo Can we get a docs review on the readme updates here please? Also, if there's a dedicated person for the Atlas Integrations, let us know and we can tag them in future.

Thanks!

**Comment by Zuhairahmed** [comment]
> @jwilliams-mongo Can we get a docs review on the readme updates here please? Also, if there's a dedicated person for the Atlas Integrations, let us know and we can tag them in future.
> 
> Thanks!

Captured as part of [DOCSP-28940](https://jira.mongodb.org/browse/DOCSP-28940)

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration
+
+
+
+## Overview
+
+![simple-quickstart-arch](https://user-images.githubusercontent.com/5663078/229119386-0dbc6e30-a060-465e-86dd-f89712b0fc49.png)
+
+The quickstart template enables you to begin working with your Machine Learning models rapidly, using your MongoDB Atlas Cluster and Amazon SageMaker Endpoint as tools. With this template, you can utilize MongoDB as a data source and Amazon SageMaker for data analysis, streamlining the process of building and deploying machine learning models.
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:9]:
```suggestion
This Partner Solutions template enables you to begin working with your Machine Learning models rapidly, using your MongoDB Atlas Cluster and Amazon SageMaker Endpoint as tools. With this template, you can utilize MongoDB as a data source and Amazon SageMaker for data analysis, streamlining the process of building and deploying machine learning models.
```

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+
+
+## MongoDB Atlas CFN Resources used by the templates
+
+- [MongoDB::Atlas::Trigger](../../cfn-resources/trigger/)
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:15]: Is this really the only MongoDB Atlas CFN resource used by this template?

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md (hunk 1 above)
Comment (Zuhairahmed) [reply examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:9]: Note AWS Quick Start have been rebranded as AWS Partner Solution Deployments: https://aws.amazon.com/quickstart/?solutions-all.sort-by=item.additionalFields.sortDate&solutions-all.sort-order=desc&awsf.filter-content-type=*all&awsf.filter-tech-category=*all&awsf.filter-industry=*all&solutions-all.q=mongodb&solutions-all.q_operator=AND

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+
+## Environment Configured by the Quickstart template
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:18]:
```suggestion
## Environment Configured by the Partner Solutions template
```

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+
+## Permissions required to run the template
+See [execution-role.yaml](execution-role.yaml) for the list of permissions needed to run the template. 
+
+
+If you want to create an IAM (Identity and Access Management) role to run the quickstart template on a CFN (CloudFormation) stack, you can use the [execution-role.yaml](execution-role.yaml) file. Note that passing an IAM role to CloudFormation when creating a stack is optional. If you don't supply one, the user permissions are assumed. 
+See the [IAM permissions section in the General information guide](https://aws-ia.github.io/content/qs_info.html#_technical_requirements) for more information.
+
+
+## Additional Information
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:35]: This is a great/helpful section @andreaangiolillo / @poornatejakonathampi !

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
+# quickstart-mongodb-atlas-mean-stack-aws-fargate-integration
+
+
+
+## Overview
+
+![simple-quickstart-arch](https://user-images.githubusercontent.com/5663078/229105149-59015114-1c14-44e3-ad5a-b48d9a487797.png)
+
+This quickstart template provides the architecture necessary to scale a MEAN (MongoDB, Express, Angular, Node.js) stack application using a combination of CloudFormation, MongoDB Atlas, and AWS Fargate. Specifically, the template leverages the MongoDB Atlas CFN resources to configure the MongoDB infrastructure and uses AWS CFN resources to set up an Application Load Balancer and a VPC. Additionally, the template employs AWS Fargate to run your Docker image.
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:9]:
```suggestion
This Partner Solutions template provides the architecture necessary to scale a [MEAN](https://www.mongodb.com/mean-stack) (MongoDB, Express, Angular, Node.js) stack application using a combination of CloudFormation, MongoDB Atlas, and AWS Fargate. Specifically, the template leverages the MongoDB Atlas CFN resources to configure the MongoDB infrastructure and uses AWS CFN resources to set up an Application Load Balancer and a VPC. Additionally, the template employs AWS Fargate to run your Docker image.
```

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+- [MongoDB::Atlas::Cluster](../../cfn-resources/cluster/)
+- [MongoDB::Atlas::ProjectIpAccessList](../../cfn-resources/project-ip-access-list/)
+- [MongoDB::Atlas::DatabaseUser](../../cfn-resources/database-user/)
+- [MongoDB::Atlas::Project](../../cfn-resources/project/)
+- [MongoDB::Atlas::NetworkPeering](../../cfn-resources/network-peering/)
+- [MongoDB::Atlas::NetworkContainer](../../cfn-resources/network-container/)
+- [MongoDB::Atlas::PrivateEndpoint](../../cfn-resources/private-endpoint/)
+
+
+## Environment Configured by the Quickstart template
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:24]:
```suggestion
## Environment Configured by the Partner Solutions template
```

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 7]
```diff
@@ -0,0 +1,52 @@
...
+The quickstart templates will generate the following resources:
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:25]:
```suggestion
The Partner Solutions templates will generate the following resources:
```

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 8]
```diff
@@ -0,0 +1,52 @@
...
+ - A virtual private cloud (VPC) configured with public and private subnets, according to AWS best practices, to provide you with your own virtual network on AWS. The VPC provides Domain Name System (DNS) resolution. The quickstart temaplate leverage the [official AWS quickstart teamplate](https://github.com/aws-quickstart/quickstart-aws-vpc/blob/9dc47510f71f1fb6baf8c4e96b5330a6f51f540e/templates/aws-vpc.template.yaml) to build your VPC infrastructure. See [Deployment Guide](https://aws-quickstart.github.io/quickstart-aws-vpc/) for more information.
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:26]:
```suggestion
 - A virtual private cloud (VPC) configured with public and private subnets, according to AWS best practices, to provide you with your own virtual network on AWS. The VPC provides Domain Name System (DNS) resolution. The template leverages the [official AWS quickstart template](https://github.com/aws-quickstart/quickstart-aws-vpc/blob/9dc47510f71f1fb6baf8c4e96b5330a6f51f540e/templates/aws-vpc.template.yaml) to build your VPC infrastructure. See [Deployment Guide](https://aws-quickstart.github.io/quickstart-aws-vpc/) for more information.
```

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+- Set up an Atlas Project within the organization (input provided).
+- Create an Atlas Cluster that has authentication and authorization enabled, and is not accessible through the public internet.
+- Create a Database user with access to the Atlas Cluster.
+- Add the IP address range 0.0.0.0/1 to the Atlas access list, allowing the cluster to be accessed through the public internet.
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:31]: Do we need to open CIDR range to whole world?

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 10]
```diff
@@ -0,0 +1,52 @@
...
+
+
+If you want to create an IAM (Identity and Access Management) role to run the quickstart template on a CFN (CloudFormation) stack, you can use the [execution-role.yaml](execution-role.yaml) file. Note that passing an IAM role to CloudFormation when creating a stack is optional. If you don't supply one, the user permissions are assumed. 
+See the [IAM permissions section in the General information guide](https://aws-ia.github.io/content/qs_info.html#_technical_requirements) for more information.
+
+
+## Additional Information
+
+- Repository: [quickstart-mongodb-atlas-mean-stack-aws-fargate-integration](https://github.com/aws-quickstart/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/tree/main)
+- Template:
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:48]: Fix formatting, should there be something after "Template"?

File: examples/quickstart-mongodb-atlas/README.md [hunk 11]
```diff
@@ -0,0 +1,67 @@
+# quickstart-mongodb-atlas
+
+
+
+## Overview
+
+![image](https://user-images.githubusercontent.com/5663078/229103723-4c6b9ab1-9492-47ba-b04d-7f29079e3817.png)
+
+The Atlas Quickstart templates allow you to set up all you need to start using MongoDB Atlas. We provide four different templates:
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas/README.md:9]:
```suggestion
The Atlas Partner Solutions templates allow you to set up all you need to start using MongoDB Atlas. We provide four different templates:
```

File: examples/quickstart-mongodb-atlas/README.md [hunk 12]
```diff
@@ -0,0 +1,67 @@
...
+- [MongoDB::Atlas::Cluster](../../cfn-resources/cluster/)
+- [MongoDB::Atlas::ProjectIpAccessList](../../cfn-resources/project-ip-access-list/)
+- [MongoDB::Atlas::DatabaseUser](../../cfn-resources/database-user/)
+- [MongoDB::Atlas::Project](../../cfn-resources/project/)
+- [MongoDB::Atlas::NetworkPeering](../../cfn-resources/network-peering/)
+- [MongoDB::Atlas::NetworkContainer](../../cfn-resources/network-container/)
+- [MongoDB::Atlas::PrivateEndpoint](../../cfn-resources/private-endpoint/)
+
+
+## Environment Configured by the Quickstart template
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas/README.md:31]:
```suggestion
## Environment Configured by the Partner Solutions template
```

File: examples/quickstart-mongodb-atlas/README.md [hunk 13]
```diff
@@ -0,0 +1,67 @@
...
+All quickstart templates will generate the following resources:
+    - An Atlas Project in the organization that was provided as input.
+    - An Atlas Cluster with authentication and authorization enabled, which cannot be accessed through the public internet.
+    - A Database user that can access the cluster.
+    - The IP address range 0.0.0.0/1 will be added to the Atlas access list, allowing the cluster to be accessed through the public internet.
```
Comment (Zuhairahmed) [inline examples/quickstart-mongodb-atlas/README.md:36]: Does CIDR range need to be open to whole world for this one too?

**Review by Zuhairahmed — APPROVED** [review]
Few comments, not-blocking. otherwise LGTM

**Review by maastha — DISMISSED** [review]
LGTM

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md (hunk 2 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:15]: yes, the template asks the user to provide ProjectId and collection and DB name. You can search for `MongoDB::Atlas::` in the [template](https://github.com/aws-quickstart/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/blob/0ca645280a474cad82443caa3bddb1c6b6c71e39/templates/mongodb-sagemaker-analytics.template.yaml#L98)

File: examples/quickstart-mongodb-atlas/README.md (hunk 13 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas/README.md]: Updated

File: examples/quickstart-mongodb-atlas/README.md (hunk 12 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas/README.md]: Updated

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md (hunk 3 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md]: Updated

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md (hunk 5 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md]: Updated

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md (hunk 6 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md]: Updated

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md (hunk 7 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md]: Updated

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md (hunk 8 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md]: Updated

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md (hunk 9 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md]: Updated

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md (hunk 10 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md]: Updated

File: examples/quickstart-mongodb-atlas/README.md (hunk 11 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas/README.md]: Updated

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md [hunk 14]
```diff
@@ -0,0 +1,41 @@
+# quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration
+
+
+
+## Overview
+
+![simple-quickstart-arch](https://user-images.githubusercontent.com/5663078/229119386-0dbc6e30-a060-465e-86dd-f89712b0fc49.png)
+
+This Partner Solutions template enables you to begin working with your Machine Learning models rapidly, using your MongoDB Atlas Cluster and Amazon SageMaker Endpoint as tools. With this template, you can utilize MongoDB as a data source and Amazon SageMaker for data analysis, streamlining the process of building and deploying machine learning models.
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:9]:
```suggestion
This Partner Solutions template enables you to begin working with your machine learning models using your MongoDB Atlas Cluster and Amazon SageMaker Endpoint as tools. With this template, you can utilize MongoDB as a data source and Amazon SageMaker for data analysis, streamlining the process of building and deploying machine learning models.
```

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md [hunk 15]
```diff
@@ -0,0 +1,41 @@
...
+- [MongoDB::Atlas::Trigger](../../cfn-resources/trigger/)
+
+
+## Environment Configured by the Partner Solutions template
+The Partner Solutions template will generate the following resources:
+ - Configure [MongoDB Partner Event Bus](http://mongodb.com/docs/atlas/app-services/triggers/aws-eventbridge/#std-label-aws-eventbridge).
+ - Set up a [database trigger](https://www.mongodb.com/docs/atlas/app-services/triggers/database-triggers/) with your Atlas Cluster.
+ - Create lambda functions to run the machine learning model and send the classification results to your MongoDB Atlas Cluster. 
+    - See [iris_classifier](sagemaker-example/iris_classifier/) for an example of machine learning model to use with this template.
+    -  See [lambda_functions](sagemaker-example/lambda_functions/) for an example of lambda functions to use to read and write data to your MongoDB Atlas cluster.
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:24]:
```suggestion
 - a [MongoDB Partner Event Bus](http://mongodb.com/docs/atlas/app-services/triggers/aws-eventbridge/#std-label-aws-eventbridge)
 - a [database trigger](https://www.mongodb.com/docs/atlas/app-services/triggers/database-triggers/) with your Atlas Cluster
 - lambda functions to run the machine learning model and send the classification results to your MongoDB Atlas Cluster. (See [iris_classifier](sagemaker-example/iris_classifier/) for an example of machine learning model to use with this template. See [lambda_functions](sagemaker-example/lambda_functions/) for an example of lambda functions to use to read and write data to your MongoDB Atlas cluster.)
```

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md [hunk 16]
```diff
@@ -0,0 +1,41 @@
...
+
+
+
+## MongoDB Atlas CFN Resources used by the templates
+
+- [MongoDB::Atlas::Trigger](../../cfn-resources/trigger/)
+
+
+## Environment Configured by the Partner Solutions template
+The Partner Solutions template will generate the following resources:
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:19]:
```suggestion
The Partner Solutions template will generate and configure the following resources:
```

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md [hunk 17]
```diff
@@ -0,0 +1,41 @@
...
+ - Create lambda functions to run the machine learning model and send the classification results to your MongoDB Atlas Cluster. 
+    - See [iris_classifier](sagemaker-example/iris_classifier/) for an example of machine learning model to use with this template.
+    -  See [lambda_functions](sagemaker-example/lambda_functions/) for an example of lambda functions to use to read and write data to your MongoDB Atlas cluster.
+
+
+## Permissions required to run the template
+See [execution-role.yaml](execution-role.yaml) for the list of permissions needed to run the template. 
+
+
+If you want to create an IAM (Identity and Access Management) role to run the quickstart template on a CFN (CloudFormation) stack, you can use the [execution-role.yaml](execution-role.yaml) file. Note that passing an IAM role to CloudFormation when creating a stack is optional. If you don't supply one, the user permissions are assumed.
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:31]: What are the default permissions? What does "user permissions" refer to?

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 18]
```diff
@@ -0,0 +1,51 @@
+# quickstart-mongodb-atlas-mean-stack-aws-fargate-integration
+
+
+
+## Overview
+
+![simple-quickstart-arch](https://user-images.githubusercontent.com/5663078/229105149-59015114-1c14-44e3-ad5a-b48d9a487797.png)
+
+This Partner Solutions template provides the architecture necessary to scale a [MEAN](https://www.mongodb.com/mean-stack) (MongoDB, Express, Angular, Node.js) stack application using a combination of CloudFormation, MongoDB Atlas, and AWS Fargate. Specifically, the template leverages the MongoDB Atlas CFN resources to configure the MongoDB infrastructure and uses AWS CFN resources to set up an Application Load Balancer and a VPC. Additionally, the template employs AWS Fargate to run your Docker image.
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:9]:
```suggestion
This Partner Solutions template provides the architecture necessary to scale a [MEAN](https://www.mongodb.com/mean-stack) (MongoDB, Express, Angular, Node.js) stack application using a combination of CloudFormation, MongoDB Atlas, and AWS Fargate. The template leverages the MongoDB Atlas CFN resources to configure the MongoDB infrastructure and AWS CFN resources to set up an Application Load Balancer and a VPC. Additionally, the template employs AWS Fargate to run your Docker image.
```

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 19]
```diff
@@ -0,0 +1,51 @@
...
+
+- Set up an Atlas Project within the organization (input provided).
+- Create an Atlas Cluster that has authentication and authorization enabled, and is not accessible through the public internet.
+- Create a Database user with access to the Atlas Cluster.
+- Add the IP address range provided as input to the Atlas access list, allowing the cluster to be accessed through the public internet.
+- Establish a VPC peering connection between the MongoDB Atlas VPC (where the cluster is located) and the VPC on AWS.
+- An application Load Balancer 
+- Set up AWS Fargate to run your Docker image. See [fargate-example/](fargate-example/) for an example of docker images to use with Fargate.
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:34]:
```suggestion
## Environment Configured by the Partner Solutions template
The Partner Solutions template will generate the following resources:
 - A virtual private cloud (VPC) configured with public and private subnets, according to AWS best practices, to provide you with your own virtual network on AWS. The VPC provides Domain Name System (DNS) resolution. The template leverages the [official AWS quickstart template](https://github.com/aws-quickstart/quickstart-aws-vpc/blob/9dc47510f71f1fb6baf8c4e96b5330a6f51f540e/templates/aws-vpc.template.yaml) to build your VPC infrastructure. See [Deployment Guide](https://aws-quickstart.github.io/quickstart-aws-vpc/) for more information.
- An Atlas Project within the organization (input provided).
- An Atlas Cluster that has authentication and authorization enabled, and is not accessible through the public internet.
- A Database user with access to the Atlas Cluster.
- Add the IP address range provided as input to the Atlas access list, allowing the cluster to be accessed through the public internet.
- A VPC peering connection between the MongoDB Atlas VPC (where the cluster is located) and the VPC on AWS.
- An application Load Balancer 
- AWS Fargate to run your Docker image. See [fargate-example/](fargate-example/) for an example of docker images to use with Fargate.
```

**Review by Zuhairahmed — APPROVED** [review]

File: examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md (hunk 17 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md:31]: > What does "user permissions" refer to?

When you log in to the AWS console, you must provide credentials for your AWS user. User permissions are those associated with your AWS user that you used to log in to the AWS console.

> What are the default permissions?

It depends on how you set up your AWS account.

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 20]
```diff
@@ -0,0 +1,51 @@
...
+
+
+## Permissions required to run the template
+See [execution-role.yaml](execution-role.yaml) for the list of permissions needed to run the template. 
+
+
+If you want to create an IAM (Identity and Access Management) role to run the Partner Solutions template on a CFN (CloudFormation) stack, you can use the [execution-role.yaml](execution-role.yaml) file. Note that passing an IAM role to CloudFormation when creating a stack is optional. If you don't supply one, the user permissions are assumed.
```
[the remark this answers is not in the document]
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:41]: I added my answer

**Comment by andreaangiolillo** [comment]
@pierwill Ready again for your review

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 21]
```diff
@@ -0,0 +1,50 @@
...
+- [MongoDB::Atlas::Project](../../cfn-resources/project/)
+- [MongoDB::Atlas::NetworkPeering](../../cfn-resources/network-peering/)
+- [MongoDB::Atlas::NetworkContainer](../../cfn-resources/network-container/)
+- [MongoDB::Atlas::PrivateEndpoint](../../cfn-resources/private-endpoint/)
+
+
+## Environment Configured by the Partner Solutions template
+The Partner Solutions template will generate the following resources:
+ - A virtual private cloud (VPC) configured with public and private subnets, according to AWS best practices, to provide you with your own virtual network on AWS. The VPC provides Domain Name System (DNS) resolution. The template leverages the [official AWS quickstart template](https://github.com/aws-quickstart/quickstart-aws-vpc/blob/9dc47510f71f1fb6baf8c4e96b5330a6f51f540e/templates/aws-vpc.template.yaml) to build your VPC infrastructure. See [Deployment Guide](https://aws-quickstart.github.io/quickstart-aws-vpc/) for more information.
+- An Atlas Project within the organization (input provided).
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:27]: What does "input provided" mean?

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 22]
```diff
@@ -0,0 +1,50 @@
...
+- An Atlas Cluster that has authentication and authorization enabled, and is not accessible through the public internet.
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:28]:
```suggestion
- An Atlas Cluster with authentication and authorization enabled, and not accessible through the public internet.
```

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md [hunk 23]
```diff
@@ -0,0 +1,50 @@
...
+- A Database user with access to the Atlas Cluster.
+- Add the IP address range provided as input to the Atlas access list, allowing the cluster to be accessed through the public internet.
```
Comment (pierwill) [inline examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md:30]:
```suggestion
- An Atlas IP access list, allowing the cluster to be accessed through the public internet.
```

File: examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md (hunk 21 above)
Comment (andreaangiolillo) [reply examples/quickstart-mongodb-atlas-mean-stack-aws-fargate-integration/README.md]: It was an error. Updated

**Review by pierwill — APPROVED** [review]
Looks good! I'm sure we'll do a more edits as we develop these docs further.

**Pull request merged**